app/containers/MassEnergizeSuperAdmin/Community/AllCommunities.js

```diff
@@ -139,40 +163,46 @@ class AllCommunities extends React.Component {
             </Link>
             &nbsp;&nbsp;
           </div>
-        )
-      }
+        ),
+      },
     },
-  ]
-
+  ];
 
   render() {
-    const title = brand.name + ' - All Communities';
+    const title = brand.name + " - All Communities";
     const description = brand.desc;
-    const { data, columns, loading } = this.state;
+    const { columns, loading } = this.state;
     const { classes } = this.props;
+    const data = this.fashionData(this.props.communities);
 
     const options = {
-      filterType: 'dropdown',
-      responsive: 'stacked',
+      filterType: "dropdown",
+      responsive: "stacked",
       print: true,
-      rowsPerPage: 100,
+      rowsPerPage: 50,
       onRowsDelete: (rowsDeleted) => {
         const idsToDelete = rowsDeleted.data;
-        idsToDelete.forEach(d => {
+        idsToDelete.forEach((d) => {
```

Comment:
I think there needs to be protection here.  A dialog saying 'are you sure you want to delete this community? (or other item).  We should also change the back end to record which admin user creates/modifies/deletes an item.

Reply:
Noted.